lib/device.js

```diff
@@ -7,6 +7,7 @@ module.exports = Device;
 function Device(bus, subnet, id) {
   this.id = parseInt(id);
   this.subnet = parseInt(subnet);
+  this.address = this.subnet + '.' + this.id;
```

Comment:
Getter function should be used here also

Reply:
Agree.